commcare_export/env.py

```diff
@@ -590,7 +594,20 @@ def lookup(self, key):
     def emit_table(self, table_spec):
         self.emitted = True
         table_spec.rows = self._unwrap_row_vals(table_spec.rows)
-        self.writer.write_table(table_spec)
+        try:
+            self.writer.write_table(table_spec)
+        except Exception as err:
+            if (
+                not logger.isEnabledFor(logging.DEBUG)  # not --verbose
+                and 'Row size too large' in str(err)
+            ):
+                logging.error(
+                    'Row size too large. You may be trying to export too many '
+                    'columns. A maximum of 200 columns is suggested.'
```

Comment:
> A maximum of 200 columns **is suggested**.

I don't know if we should make it a suggestion, as this makes it sound like the user can still go a little above this limit. Given we have a hard-set limit, it might be better to phrase this as something like "A maximum of 200 columns is supported."

Reply:
This should (I hope) explain the error better, and frame the 200-column guideline: 531df4b78651321edaed0de243467a21fa2a8cb2

Reply:
It sounds great, thanks for the rework!